examples/fabric-admin/commands/pairing/PairingCommand.cpp

```diff
@@ -389,7 +393,11 @@ void PairingCommand::OnCommissioningComplete(NodeId nodeId, CHIP_ERROR err)
 {
     if (err == CHIP_NO_ERROR)
     {
-        ChipLogProgress(NotSpecified, "Device commissioning completed with success");
+        fprintf(stderr, "New device with Node ID: 0x%lx has been successfully added.\n", nodeId);
```

Comment:
Why fprintf ? Why not ChipLogProcess?

Reply:
Because I need this message to get printed out to console instead of log.